src/DynamoCore/Models/NodeModel.cs

```diff
@@ -1476,6 +1482,32 @@ protected virtual ExecutionHints GetExecutionHintsCore()
             return executionHint;
         }
 
+
+        public bool ShowExecutionPreview
```

Comment:
I want `ShowExecutionPreview` in the view model layer. This property has no logical reason for being in the model.